com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/helpers/DummyConfigurationFilterParser.java

```diff
@@ -8,7 +8,7 @@
     private final ConfigurationFilter filter;
 
     public DummyConfigurationFilterParser(ConfigurationFilter filter) {
-        super(null, null);
+        super(null, null, null);
```

Comment:
Can we add empty constuctor in super class?

Reply:
I think it's more explicit this way that the null values aren't actually a default behaviour